app/router.js

```diff
@@ -61,6 +61,7 @@ Router.map(function () {
     this.route('build', { path: '/builds/:build_id', resetNamespace: true }, function () {
       this.route('config');
     });
+    this.route('config');
```

Comment:
I think we have discussed the name of this route previously, but didn't come to a conclusion, right? Should we continue the discussion here?

Reply:
yep, that's right. i've taken a note about this on this PR in the description above (the original discussion was [here](https://github.com/travis-ci/travis-web/pull/2405#discussion_r396682366). we can continue this discussion here, although i'm concerned others might not see it, but maybe that's alright, too.

the candidates mentioned so far are:

* `repo/:provider/:slug/config`
* `repo/:provider/:slug/request/config`
* `repo/:provider/:slug/build/config`
* `repo/:provider/:slug/requests/new/config`

i have given this more thought over the last two weeks, and i think we should stick with the current route name `repo/:provider/:slug/config` for these reasons:

* throughout our entire system "config" represents the _build config_ as provided by the user in their `.travis.yml` file, any config imports, and build config posted via API. for any other configuration we use the term "settings". this is the case in our database layout, internal model/service/etc naming, on our public API v2 and v3, and in reflected in the UI (as far as i can tell the only place "config" is being used visibly is on the build config tab.

  i think this addresses the concern of `repo/config` potentially being too overloaded in the future. i do not see us ever adding another concept of "repo config" which would not relate to the user's build config. in other words, i think in our system and product "config" is a synonymous with "build config".

* the user's build configuration that we accept represents an entire user interface on its own that has a significant complexity. this is, so far, not actually reflected in our UI (i suppose for the simple reason that it has been too complex to address, so far). with travis-yml in place we can now adjust that a little, and give build configs more of a first class visibility ... a proper "home" in our UI.
 
  i think this addresses the concern that `repo/config` is a rather top level, generic, not very much scoped naming. for such an important concern as the user's build config interface that seems alright to me. 

* the semantical mapping `repo <> config` directly matches the user's experience of storing a build config (their `.travis.yml`) within their git repository, which seems to make this a good choice, too.

* namings involving an action (such as "trigger build" or "new build request") seem to reduce the scope from an entity to an action, which makes them seem less fitting to me.

* naming involving `request` bring up the old dissent whether or not requests represent an internal implementation detail in our system. the argument made was that users care about builds, and can get confused by the term (build) request. while our API accepts build requests on the route `POST repo/:id/request` and i personally tend to disagree with the whole notion (i'd like requests to be promoted to be more first class citizens in the UI, too) ... i'd probably avoid that whole rabbit hole, and thus avoid the term `request` in a user facing route.

so, in conclusion ... i think `repo/:provider/:slug/config` is a great choice. the only other choice i think i would make would be `repo/:provider/:slug/build{s}/config`, but i don't really see a huge benefit added by mentioning `build`, too.

Reply:
@svenfuchs thank you for such a detailed explanation! I totally agree with all of your points, and especially giving it another thought, I feel as well that `config = .travis.yml` and it belongs to `repo` directly, so it seems very natural and intuitive 👍

Reply:
The route and reasoning makes sense to me :+1: